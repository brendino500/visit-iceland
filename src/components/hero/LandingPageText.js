import { Typography } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles((theme) => ({
  container: {
    display: 'flex',
  },
  title: {
    fontSize: 60,
    textTransform: 'uppercase',
    fontFamiy: 'Barlow',
    letterSpacing: 3,
    textAlign: 'right',
  },
}))

export default function LandingPageText() {
  const classes = useStyles()
  return (
    <div className={classes.container}>
      <Typography className={classes.title}>Visit Iceland</Typography>
    </div>
  )
}
