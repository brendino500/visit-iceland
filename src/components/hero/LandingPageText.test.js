import renderer from 'react-test-renderer'
import LandingPageText from './LandingPageText'

it('LandingPageText matches snapshot', () => {
  const tree = renderer.create(<LandingPageText />).toJSON()
  expect(tree).toMatchSnapshot()
})
