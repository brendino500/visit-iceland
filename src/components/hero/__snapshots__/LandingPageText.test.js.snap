// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPageText matches snapshot 1`] = `
<div
  className="container mx-auto flex content-center justify-end "
>
  <div
    className="z-40"
  >
    <h1
      className="text-7xl uppercase text-gray-300 font-serif content-center mt-48 mr-80 "
    >
      Visit Iceland
    </h1>
  </div>
</div>
`;
